spring-javaformat-eclipse/io.spring.javaformat.eclipse/META-INF/MANIFEST.MF

```diff
@@ -16,7 +16,7 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.m2e.jdt;resolution:=optional,
  org.eclipse.m2e.core;resolution:=optional,
  org.eclipse.m2e.maven.runtime;resolution:=optional,
- org.eclipse.buildship.core;bundle-version="[3.0.0,3.1.0)";resolution:=optional,
+ org.eclipse.buildship.core;bundle-version="[3.0.0,3.3.0)";resolution:=optional,
```

Comment:
Given the switch from `Set` to `Collection` below, I suspect there has been some breaking API changes in Buildship. Consequently, should the bottom end of this range be increased?

Reply:
We're good to keep the lower bound as is. The `ModelProviderUtil` class used prevuously is gone. Instead I'm using different API to get the same thing really: `InternalGradleBuild#getModelProvider()` and `ModelProvider#fetchModels(Class<T> model, FetchStrategy strategy, CancellationTokenSource tokenSource, IProgressMonitor monitor)`
This API is available in 3.0 as well from I'm seeing in the git repo for 3.0.0 release tag:
https://github.com/eclipse/buildship/blob/REL_3.0.0/org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/internal/workspace/ModelProvider.java
https://github.com/eclipse/buildship/blob/REL_3.0.0/org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/internal/workspace/InternalGradleBuild.java#L31